src/BloomExe/web/ServerBase.cs

```diff
@@ -340,11 +340,13 @@ protected internal static string CorrectedLocalPath(IRequestInfo info)
 		{
 			// Note that LocalPathWithoutQuery removes all % escaping from the URL.
 			var result = info.LocalPathWithoutQuery;
-			if (info.RawUrl.StartsWith(BloomUrlPrefix + "//"))
+			var idx = info.RawUrl.IndexOf("///");
```

Comment:
Just reading this over, I can't tell if it's correct or not, or if it handles the case of the previous version (looks quite different: '///' vs. '//', start with vs starts *after* the first character (in other words starts a 0 vs. starts *after* 0). 

Seems like we need a set of unit tests. THis could become public, it's just a static, and then easily testable.